src/html/html-creator/element.js

```diff
@@ -18,22 +20,9 @@ const parseContent = (content, createElementMethod) => {
     if (content && content.constructor === Array) {
         return content.map(element => createElementMethod(element)).join('');
     }
-    return escapeHtmlEntities(content) || '';
+    return htmlEntities.escape(content) || '';
```

Comment:
How does it handle undefined?

```suggestion
    return content ? htmlEntities.escape(content) : '';
```

Reply:
It's already handled in `create()`.  So basically `parseContent()` will never receive undefined as content.